### PR TITLE
Ledger submodule

### PR DESCRIPTION
**Pull request by prokopsimek**
- created `tatum-ledger` submodule
- refactored `tatum-ada` to use new submodule

Questions:
- Should I remove all re-exported ledger functions in the ada submodule?
- Is that fine to have just `createAccount` function in `tatum-ada` submodule?

TODO:
- refactor all other chains

### CODE REVIEW

File: packages/tatum-ada/package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@emurgo/cardano-serialization-lib-nodejs": "^6.0.0",
     "@tatumio/tatum-core": "file:../tatum-core",
+    "@tatumio/tatum-ledger": "file:../tatum-ledger",
```
Comment (prokopsimek) [inline packages/tatum-ada/package.json:18]: info: `createAccountCore` functions has been moved to `tatum-ledger` submodule

File: packages/tatum-ada/src/ledger/account.ts [hunk 2]
```diff
@@ -30,4 +31,4 @@ export {
   getAccountsByCustomerId,
   getAllAccounts,
   getAccountBalance,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```
Comment (prokopsimek) [inline packages/tatum-ada/src/ledger/account.ts:34]: Should I remove these re-exports or is that necessary to have them here in specific chain module?

File: packages/tatum-ada/src/ledger/customer.ts [hunk 3]
```diff
@@ -6,4 +6,4 @@ export {
   deactivateCustomer,
   enableCustomer,
   disableCustomer,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```
Comment (prokopsimek) [inline packages/tatum-ada/src/ledger/customer.ts:9]: Should I remove these re-exports or is that necessary to have them here in specific chain module?

File: packages/tatum-ada/src/ledger/orderBook.ts [hunk 4]
```diff
@@ -6,4 +6,4 @@ export {
   getTradeById,
   deleteTrade,
   deleteAccountTrades,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```
Comment (prokopsimek) [inline packages/tatum-ada/src/ledger/orderBook.ts:9]: Should I remove these re-exports or is that necessary to have them here in specific chain module?

File: packages/tatum-ada/src/ledger/subscription.ts [hunk 5]
```diff
@@ -3,4 +3,4 @@ export {
   listActiveSubscriptions,
   cancelExistingSubscription,
   obtainReportForSubscription,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```
Comment (prokopsimek) [inline packages/tatum-ada/src/ledger/subscription.ts:6]: Should I remove these re-exports or is that necessary to have them here in specific chain module?

File: packages/tatum-ada/src/ledger/transaction.ts [hunk 6]
```diff
@@ -7,4 +7,4 @@ export {
   countTransactionsByAccount,
   countTransactionsByCustomer,
   countTransactionsByLedger,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```
Comment (prokopsimek) [inline packages/tatum-ada/src/ledger/transaction.ts:10]: Should I remove these re-exports or is that necessary to have them here in specific chain module?

File: packages/tatum-ada/src/ledger/vc.ts [hunk 7]
```diff
@@ -4,4 +4,4 @@ export {
   updateVirtualCurrency,
   mintVirtualCurrency,
   revokeVirtualCurrency,
-} from '@tatumio/tatum-core'
+} from '@tatumio/tatum-ledger'
```
Comment (prokopsimek) [inline packages/tatum-ada/src/ledger/vc.ts:7]: Should I remove these re-exports or is that necessary to have them here in specific chain module?

File: packages/tatum-ada/src/ledger/customer.ts (hunk 3 above)
Comment (samuelsramko) [reply packages/tatum-ada/src/ledger/customer.ts:9]: We should keep reexports per chain

File: packages/tatum-ada/src/ledger/subscription.ts (hunk 5 above)
Comment (samuelsramko) [reply packages/tatum-ada/src/ledger/subscription.ts:6]: Keep

**Review by samuelsramko — APPROVED** [review]

**Pull request merged**